data/transit/route/bus.json

```diff
@@ -2750,17 +2750,23 @@
         "route": "bus"
       }
     },
+    {
+      "displayName": "CH-GR",
+      "id": "chgr-ba16da",
+      "locationSet": {"include": ["ch"]},
+      "tags": {"network": "CH-GR", "route": "bus"}
+    },
     {
       "displayName": "CH-UR",
       "id": "chur-ba16da",
       "locationSet": {"include": ["ch"]},
-      "tags": {
-        "network": "CH-UR",
-        "operator": "Auto AG Uri",
-        "operator:short": "AAGU",
-        "operator:wikidata": "Q33058015",
-        "route": "bus"
-      }
+      "tags": {"network": "CH-UR", "route": "bus"}
+    },
+    {
+      "displayName": "CH-VS",
+      "id": "chvs-ba16da",
```

Comment:
> At first no changes, only a dry run using the release checklist, to rebuild the index.

Hello, thank you for contributing to NSI, and this is not required when open a PR because this checklist is for maintainer when they publish a new npm version.

---

However, a entry without wikidata won't be suggested in editor, see:

https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required

Reply:
I see. I don't expect, that there will be a (valid) wikidata entry for those pseudo-networks. Should they be removed then? Because CH-UR has no wikidata entry and it's incorrect, that there's only one operator. Also KtGR (should actually be CH-GR) has no wikidata entry. Suggestions?